README.md

```diff
@@ -80,33 +80,27 @@ filter, or even split the same type of filter over multiple files. Ultimately, a
 your `params.ini` will be used to determine if an item should be kept. If one of the profiles wants to keep the item, it
 will be kept regardless of the other profiles. Similarly, if a filter is missing in all profiles (e.g., there is no `Sigils` section in any profile), all corresponding items (in this case, sigils) will be kept.
 
-### Aspects
+### Affix / Aspects Filter Syntax
```

Comment:
Its only Affix filtering, no more aspects, no?

Reply:
Can be used for unique items